backend/danswer/llm/utils.py

```diff
@@ -56,16 +56,26 @@ def get_default_llm_token_encode() -> Callable[[str], Any]:
     return _LLM_TOKENIZER_ENCODE
 
 
+def tokenizer_trim_content(
```

Comment:
just a thought, is it better to drop the last word if it doesn't have some punctuation so that we don't risk ending with a partial word?